src/cheats.c

```diff
@@ -214,25 +220,65 @@ static void cheat_10_minute_inspections()
 	window_invalidate_by_class(WC_RIDE);
 }
 
-static void cheat_increase_money(money32 amount)
+static void cheat_no_money(int enabled)
 {
-	money32 currentMoney;
+	if (enabled == 1) {
+		gParkFlags |= PARK_FLAGS_NO_MONEY;
```

Comment:
the park parameters arent synced over network yet, those are part of the debug stuff if i'm right. that would take having to start building onto making the debug stuff syncable over network, which i'm uneasy about starting up. i'm okay with making the cheat functions because they're pretty straightforward.